diesel_migrations/src/lib.rs

```diff
@@ -17,7 +78,52 @@ extern crate migrations_macros;
 #[doc(hidden)]
 pub use migrations_macros::*;
 #[doc(inline)]
-pub use migrations_internals::*;
+pub use migrations_internals::Migration;
+#[doc(inline)]
+pub use migrations_internals::MigrationName;
+#[doc(inline)]
+pub use migrations_internals::MigrationConnection;
+#[doc(inline)]
+pub use migrations_internals::MigrationError;
+#[doc(inline)]
+pub use migrations_internals::RunMigrationsError;
+#[doc(inline)]
+pub use migrations_internals::run_migration_with_version;
+#[doc(inline)]
+pub use migrations_internals::run_migrations;
+#[doc(inline)]
+pub use migrations_internals::run_pending_migrations;
+#[doc(inline)]
+pub use migrations_internals::run_pending_migrations_in_directory;
+#[doc(inline)]
+pub use migrations_internals::revert_latest_migration;
+#[doc(inline)]
+pub use migrations_internals::revert_latest_migration_in_directory;
+#[doc(inline)]
+pub use migrations_internals::revert_migration_with_version;
+#[doc(inline)]
+pub use migrations_internals::mark_migrations_in_directory;
+#[doc(inline)]
+pub use migrations_internals::any_pending_migrations;
+#[doc(inline)]
+pub use migrations_internals::setup_database;
+#[doc(inline)]
+pub use migrations_internals::migration_paths_in_directory;
+#[doc(inline)]
+pub use migrations_internals::migration_from;
+#[doc(inline)]
+pub use migrations_internals::find_migrations_directory;
+#[doc(inline)]
+pub use migrations_internals::search_for_migrations_directory;
+#[doc(inline)]
+pub use migrations_internals::version_from_path;
```

Comment:
docs are hidden for [this function](https://github.com/diesel-rs/diesel/blob/master/diesel_migrations/migrations_internals/src/migration.rs#L113) Does it still need to be inlined?